common/src/test/java/org/opensearch/ml/common/agent/MLToolSpecTest.java

```diff
@@ -50,6 +109,28 @@ public void toXContent() throws IOException {
 
     @Test
     public void parse() throws IOException {
+        String jsonStr =
+            "{\"type\":\"test\",\"name\":\"test\",\"description\":\"test\",\"parameters\":{\"test\":\"test\"},\"include_output_in_agent_response\":false,\"config\":{\"configKey\":\"configValue\"}}";
+        XContentParser parser = XContentType.JSON
+            .xContent()
+            .createParser(
+                new NamedXContentRegistry(new SearchModule(Settings.EMPTY, Collections.emptyList()).getNamedXContents()),
+                null,
+                jsonStr
+            );
+        parser.nextToken();
+        MLToolSpec spec = MLToolSpec.parse(parser);
+
+        Assert.assertEquals(spec.getType(), "test");
+        Assert.assertEquals(spec.getName(), "test");
+        Assert.assertEquals(spec.getDescription(), "test");
```

Comment:
Nit: using the same string everywhere can let copy-paste-and-forget-to-change errors slip through.  Recommend using a different string for each field.

Reply:
done